libs/neuron-ui-demo.ipynb

```diff
@@ -6,19 +6,7 @@
    "metadata": {
```

Comment:
@tarelli This file shouldn't be committed as the only changes are the python version and the output of a cell.
Re your comment, I don't want to git ignore it because we may want to change the current comment of the cell for this release.